pyproject.toml

```diff
@@ -65,6 +61,11 @@ check_done = "check_done.command:main"
 "Changelog" = "https://github.com/siisurit/check_done/blob/main/CHANGELOG.md"
 "Issue Tracker" = "https://github.com/siisurit/check_done/issues"
 
+[tool.poetry.group.dev.dependencies]
+cryptography = "^44.0.0"
+requests-mock = "^1.12.1"
+python-dotenv = "^1.0.1"
+
```

Comment:
`coverage`, `pytest`, `pytest-cov`, `pre-commit` and `ruff` should have been migrated from the old-style dev dependencies group to the new one here, and the old dev dependencies section should have been as a result removed.